docs/user_needs/index.sdoc

```diff
@@ -0,0 +1,15 @@
+[DOCUMENT]
+TITLE: Zephyr User Needs
+REQ_PREFIX: ZEP-
+
+[GRAMMAR]
+IMPORT_FROM_FILE: user_needs.sgra
+
+[FREETEXT]
+
+TBD
+
+[/FREETEXT]
+
+[DOCUMENT_FROM_FILE]
+FILE: zephyr_rtos_user.sdoc
```

Comment:
These needs are all independent of the Zephyr Project and exist before considering any RTOS. Rename `zephyr_rtos_user.sdoc` to files representing the specific user (e.g., `chip_vendor.sdoc`, `product_manufacturer.sdoc`, `student.sdoc`).